### PR TITLE
[strings] Update strings.txt EU

### PR DESCRIPTION
**Pull request by garaolaza**
Some translation corrections in the "eu" version

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Thank you!

File: data/strings/strings.txt [hunk 1]
```diff
@@ -28746,7 +28746,8 @@
     de = Die OpenStreetMap-Autorisierung ist aufgrund neuer Einschränkungen in Google Play vorübergehend nicht verfügbar 😾\nUnser Team arbeitet daran, dieses Problem zu beheben. Bitte habe etwas Geduld 🙏
     el = Η εξουσιοδότηση του OpenStreetMap δεν είναι προσωρινά διαθέσιμη λόγω νέων περιορισμών στο Google Play 😾\nΗ ομάδα μας εργάζεται για την επίλυση αυτού του προβλήματος. Παρακαλούμε να είστε υπομονετικοί 🙏
     es = La autorización de OpenStreetMap no está disponible temporalmente debido a nuevas restricciones en Google Play 😾🙏nNuestro equipo está trabajando para resolver este problema. Por favor, ten paciencia 🙏
-    et = OpenStreetMapi autoriseerimine on ajutiselt kättesaamatu Google Play uute piirangute tõttu 😾\nMeie meeskond töötab selle probleemi lahendamise kallal. Palun olge kannatlik 🙏
+    et = OpenStreetMapi autoriseerimine on ajutiselt kättesaamatu Google Play uute piirangute tõttu 😾\nMeie meeskond töötab selle probleemi lahendamise kallal. Palun olge kannatlik 🙏
```
Comment (biodranik) [inline data/strings/strings.txt:28749]: What's changed here? Why the check fails?

**Comment by Jean-BaptisteC** [comment]
@garaolaza can you remove whitespaces?

File: data/strings/strings.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (biodranik) [reply data/strings/strings.txt:28749]: I have the same issue on most PRs.

Comment (garaolaza) [reply data/strings/strings.txt:28749]: Done

**Review by garaolaza — COMMENTED** [review]
Signed-off-by: Gari Araoalza <garaolaza@codesyntax.com>

**Comment by garaolaza** [comment]
Sorry, I did not receive the notifications on your comments.

I am afraid I need some help to contribute these changes, as I am not familiar with PR's.

**Review by biodranik — APPROVED** [review]
Thanks, you did it well!

**Comment by biodranik** [comment]
Looks like there is a merge conflict, as the file has been already changed. Let me help you.

**Comment by biodranik** [comment]
Thank you for your contribution and welcome to the club )

**Comment by garaolaza** [comment]
Thank you @biodranik !  I love Organic Maps and I would like to help.

**Pull request merged**